### PR TITLE
Buttons refactor

### PR DESCRIPTION
**Pull request by lizonp**
I ahve added themming, inlining svgs before and after text, keyboard only focus and unified type selection.
secondary-theme-white
tab-focus-and-hover-secondary
primary-hover
keyboard-focus-primary
primary

### CODE REVIEW

**Comment by banciur** [comment]
- According to styleguide button primary should be 15px not 14.
- remove .btn classes from overrides.scss

File: app/components/shared/Buttons.module.scss [hunk 1]
```diff
@@ -1,58 +1,129 @@
...
+
+  .content {
+    align-items: center;
+    display: flex;
+    flex-shrink: 0;
+    justify-content: center;
+    padding-bottom: px-to-rem(8px);
+    padding-top: px-to-rem(8px);
+    padding-right: px-to-rem(20px);
+    padding-left: px-to-rem(20px);
```
[the remark this answers is not in the document]
Comment (lizonp) [reply app/components/shared/Buttons.module.scss]: 👍

File: app/components/shared/Buttons.tsx [hunk 2]
```diff
@@ -1,32 +1,57 @@
...
-    {children}
-  </Button>
-);
+interface IButtonProps {
+  layout?: TButtonLayout;
+  theme?: TButtonTheme;
+  disabled?: boolean;
+  onClick?: React.MouseEventHandler<any>;
+  svgIcon?: string;
+  to?: string;
```
[the remark this answers is not in the document]
Comment (lizonp) [reply app/components/shared/Buttons.tsx]: 👍

**Comment by huesforalice** [comment]
@krzkaczor fixed the kyc buttons and merged master

**Comment by huesforalice** [comment]
buttons

**Pull request merged**